students/ws/users/views.py

```diff
@@ -1,3 +1,49 @@
-from django.shortcuts import render
+import re
+import json
 
-# Create your views here.
\ No newline at end of file
+from django.http import JsonResponse
+from django.views import View
+from django.core.exceptions import ValidationError
```

Comment:
import 기준으로 정렬해 주세요.
```suggestion
from django.http            import JsonResponse
from django.views           import View  
from django.core.exceptions import ValidationError
```